src/reducers/events.js

```diff
@@ -0,0 +1,23 @@
+const eventsReducerDefaultState = [];
+
+const eventsReducer = (state = eventsReducerDefaultState, action) => {
+  switch (action.type) {
+    case "ADD_EVENT":
+      return [...state, action.event];
+    case "EDIT_EVENT":
+      return state.map(event => {
+        if (event.id === action.id) {
```

Comment:
Oops, I think that you should also cover the situation when `event.id !== action.id`. In other case you will remove all other events from state ;)